docs/reference/api/docker_remote_api_v1.24.md

```diff
@@ -2952,11 +2954,11 @@ Create a volume
       "Name": "tardis",
       "Driver": "local",
       "Mountpoint": "/var/lib/docker/volumes/tardis",
-      "Status": null,
```

Comment:
Thanks for your advice @thaJeztah 
Actually, I have not experienced a non local driver. I have idea about what are the response's fields, mostly on the the field `Scope`.

In addition. sorry for my not following your suggestion on `JSON fields` in the second note. Can you explain more about that? Thanks.